README.TXT

```diff
@@ -9,7 +9,7 @@ project and make sure that it actually works.  Projects tends to rot if
 you leave it alone for a few years, and it takes effort for someone to
 deal with it again.
 
-The bad news:  this code only compiles and runs on linux.  We couldn't
+The good news:  this code only compiles and runs on linux.  We couldn't
```

Comment:
in that case, change the whole line to:

The  good news:  this code compiles and runs on linux.

Reply:
After some after thought, the README should contain things like how you get it up and running, I suggest instead reverting this change and renaming the file to FOREWORD.TXT